### ISSUE TITLE
Nyxt icon is not appearing on my ubuntu desktop gnome UI while the program is running

### ISSUE THREAD

**Issue by pdelfino** [atlas-engineer/nyxt#1508]
**Describe the bug**

A picture is worth a thousand words: first [print](https://raw.githubusercontent.com/pdelfino/public-notes/main/Screenshot%20from%202021-06-15%2009-17-17.png) and [second](https://raw.githubusercontent.com/pdelfino/public-notes/main/Screenshot%20from%202021-06-15%2009-17-32.png) PrintScreen.

**Precise recipe to reproduce the issue**

1. Install Nyxt using a .deb package in ubuntu. The logo is perfectly shown while Nyxt is running;
2. Follow the developer manual and run nyxt from source. The logo is perfectly shown while Nyxt from .deb is running, while Nyxt from source is running and while there are two windows with both versions running simultaneously;
3. Remove the stable version installed with a .deb package but keep everything set to run Nyxt from source.
4. Now, having only one option, run Nyxt from source. The will **not** be showing anymore.

 **OS name+version:**
    Ubuntu 20.04.2 LTS

**Graphics card and driver:**
    product: Iris Plus Graphics 640
    configuration: driver=i915 latency=0

**Desktop environment / Window manager name+version:**
    Name: GNOME Shell 3.36.7
    Window manager's "showing the desktop" mode: OFF

**How you installed Nyxt (Guix pack, package manager, build from source):**
    Initially, I installed Nyxt version-2-pre-release-7 using a package manager (and the icon used to work). Then, I installed advanced versions from the master branch using the build from source approach and pulling the newest version from github. While having both versions installed the icon was working. Then when I removed the installation from the package manager the icon stopped working while I was running the program from the source.

  **Information from command copy-system-information:**
    Nyxt version: version 2.0.0-102-gd751ea9b
    Renderer: GI-GTK
    Operating system kernel: Linux 5.8.0-53-generic
    Lisp implementation: SBCL 2.1.0.0.HEAD.1-46cb065
    Features: (WEBKIT2 WEBKIT2-2.30 WEBKIT2-TRACKING WEBKIT2-MUTE WEBKIT2-EMOJI
    WEBKIT2-MEDIA WEBKIT2-SANDBOXING GTK-3-22 GTK-3-20 GTK-3-18 GTK-3-16
    GTK-3-14 GTK-3-12 GTK-3-10 GTK-3-8 GTK-3-6 GTK-3-4 GTK GDK-3-22
    GDK-3-20 GDK-3-18 GDK-3-16 GDK-3-14 GDK-3-12 GDK-3-10 GDK-3-8
    GDK-3-6 GDK-3-4 CAIRO-1-10 CAIRO-1-12 GDK-PIXBUF GLIB-2-30 GLIB-2-32
    GLIB-2-34 GLIB-2-36 GLIB-2-38 GLIB-2-40 GLIB-2-42 GLIB-2-44
    GLIB-2-46 GLIB-2-48 GLIB-2-50 GLIB-2-52 GLIB-2-54 GLIB-2-56
    GLIB-2-58 GLIB FSET-EXT-STRINGS CUSTOM-HASH-TABLE-NATIVE SWANK
    PLUMP-UTF-32 GLOBAL-VARS DECLARE-TYPES PARENSCRIPT NAMED-READTABLES
    21BIT-CHARS CHUNGA FLEXI-STREAMS CLOSER-MOP CL-PPCRE-UNICODE
    CL-UNICODE BORDEAUX-THREADS SPLIT-SEQUENCE GRAY-STREAMS FAST-IO-SV
    FAST-IO SBCL-USES-SB-ROTATE-BYTE ASDF-SYSTEM-CONNECTIONS
    CL-JSON-CLOS CL-JSON THREAD-SUPPORT CL-PPCRE FLAT-NAMESPACE X86-64
    UNIX CFFI FLAT-NAMESPACE SEQUENCE-EMPTYP QUICKLISP ASDF3.3 ASDF3.2
    ASDF3.1 ASDF3 ASDF2 ASDF OS-UNIX NON-BASE-CHARS-EXIST-P ASDF-UNICODE
    X86-64 GENCGC 64-BIT ANSI-CL COMMON-LISP ELF IEEE-FLOATING-POINT
    LINUX LITTLE-ENDIAN PACKAGE-LOCAL-NICKNAMES SB-CORE-COMPRESSION
    SB-LDB SB-PACKAGE-LOCKS SB-THREAD SB-UNICODE SBCL UNIX)

**Output when started from a shell**

I do not start Nyxt from a shell. I start it inside the SLIME REPL with:
```
CL-USER> (nyxt:start)
<INFO> [09:01:42] Listening to socket "/run/user/1000/nyxt/nyxt.socket".
Nyxt version 2.0.0-102-gd751ea9b
<INFO> [09:01:42] Loading Lisp file "/home/pedro/.config/nyxt/auto-config.lisp".
<INFO> [09:01:42] Loading Lisp file "/home/pedro/.config/nyxt/init.lisp".
NIL
```

**Comment by pdelfino** [comment]
Sorry about the recipe. I will read more bug reports to achieve the correct discourse. I updated the comment above. Is it good now?

It used to work with .deb installed. While having .deb version installed, it also worked while running from source.
After uninstalling the version from .deb, it does not work when I run from source.

Thus, if I got your comment right, it is expected and not a bug. Thanks! 

I think I can close the issue.
